### DISCUSSION TITLE
How to pass on original JWT token to other backends

### DISCUSSION THREAD

**Discussion by tilothiele** [nextauthjs/next-auth#11333] [Help]
Hello guys,
in the past I used pure client side React and keycloakjs in order to authenticate against the java-backend. It works very well.
I wonder how to make this work with nextjs/nextauth5.
I set up a nextjs/nextauth5/keycloak. Authentication works in this world.
I need to invoke the java-backend from the nextjs server side.
How can I pass on the JWT-Token returned from keycloak from the browser to the java backend?

**Comment by saujanya01** [comment]
You can attach the access_token to session object in sessions callback. By this way you will get the access token on client side and you can pass it to other backends.
